rmc/utils/generic.py

```diff
@@ -27,22 +27,21 @@
 )
 from rmc.resources.gnomad import constraint_ht, mutation_rate
 from rmc.resources.reference_data import (
-    asc_ssc_spark_de_novo_tsv,
-    ddd_autism_de_novo_tsv,
+    autism_de_novo_2022_tsv_path,
     dosage_ht,
-    dosage_tsv,
+    dosage_tsv_path,
     clinvar,
-    clinvar_plp_hi_mis,
-    clinvar_plp_trip_mis,
-    de_novo,
-    haplo_genes_he,
-    triplo_genes_he,
+    clinvar_plp_mis_haplo,
+    clinvar_plp_mis_triplo,
+    haplo_genes_path,
+    ndd_de_novo,
+    ndd_de_novo_2020_tsv_path,
+    triplo_genes_path,
```

Comment:
Just noticed these aren't in alphabetical order

Reply:
good point! these will also be automatically fixed in the future when https://github.com/broadinstitute/regional_missense_constraint/pull/270 is merged